ui/pages/confirmations/hooks/useGasFeeInputs.js

```diff
@@ -130,7 +132,7 @@ export function useGasFeeInputs(
     gasFeeEstimates,
     isGasEstimatesLoading,
     isNetworkBusy,
-  } = useGasFeeEstimates(transaction?.networkClientId);
```

Comment:
The cause of this bug is that we've already processed a transaction with network ID X, which is included in the transaction payload. Later, we delete the network and add a new one with ID Y. However, the system still tries to use the original ID X, which no longer exists.

Reply:
In addition to deleting the entire network, the bug is also triggered when removing just the endpoint used for a previous transaction.

Reply:
Nice fix!

Would this bug have happened for custom RPCs (where the networkClientId is a uuid)?

Also just want to confirm that the `useGasFeeEstimates` hook can handle if it's params is undefined (e.g. the network was outright deleted, very unlikely in this flow?)